plugins/generator-1.16.5/forge-1.16.5/templates/block.java.ftl

```diff
@@ -429,17 +453,6 @@ public class ${name}Block extends ${JavaModName}Elements.ModElement {
 		</#if>
 
         <#if data.isWaterloggable>
-            <#if data.rotationMode == 0>
-            @Override
-            public BlockState getStateForPlacement(BlockItemUseContext context) {

```

Comment:
This method is now missing from waterloggable blocks with no rotation (Same for 1.18)

Reply:
Doesn't it get added on the line 359?

Reply:
Tested it manually and it doesn't. The method only appears inside a `if data.rotationMode != 0`

Reply:
Oh well, I thought it was about `createBlockStateDefinition()`...

Reply:
:heavy_check_mark: Fixed